src/Command.js

```diff
@@ -226,3 +226,24 @@ export default class Command {
     throw new Error("command.execute() needs to be implemented.");
   }
 }
+
+export function commandNameFromClassName(className) {
+  return className.replace("Command", "").toLowerCase();
```

Comment:
what happens if we have `CommandCommand`? 😄 

`return className.replace(/Command$/, '').toLowerCase();`?